src/main.ts

```diff
@@ -1,8 +1,10 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 
+
 async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
+  const app = await NestFactory.create(AppModule, { cors: true });
+
   await app.listen(3030);
```

Comment:
Fetch the port number as process.env.PORT_NUMBER || PORT_NUMBER from constant file.